packages/ariakit-react-core/src/composite/composite-item.tsx

```diff
@@ -194,6 +195,7 @@ export const useCompositeItem = createHook<CompositeItemOptions>(
           id: id || item.id,
           rowId,
           disabled: !!trulyDisabled,
+          typeaheadText: typeaheadText,
```

Comment:
`typeaheadText` is missing in the callback deps. We should likely add the linter to the project as it could help identify similar issues.

Reply:
Yeah, I agree.

Could we include this lint in the following PR? I've done a few tests, and there are some things we must address in other files too.

Reply:
Yes, it's better as a separate PR.